### PR TITLE
[license] Fix typo in contributing.md

### PR DESCRIPTION
**Pull request by abiogenesis-now**
@ncdc PTAL

Signed-off-by: Jessica Yao <jessica@heptio.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,10 +3,9 @@
 
 All authors to the project retain copyright to their work. However, to ensure
 that they are only submitting work that they have rights to, we are requiring
-everyone to acknowldge this by signing their work.
+everyone to acknowledge this by signing their work.
 
-Any copyright notices in this repos should specify the authors as "The
-heptio/aws-quickstart authors".
+Any copyright notices in this repos should be specified as © 2017 Heptio Inc.
```
Comment (ncdc) [inline CONTRIBUTING.md]: repos->repo

Comment (abiogenesis-now) [reply CONTRIBUTING.md]: ah :+1:

Comment (ncdc) [reply CONTRIBUTING.md]: Please remove the year

**Comment by abiogenesis-now** [comment]
updated, let me know if this wording works

**Pull request merged**